example-soap-rest-assured/src/test/java/com/estefet/qapractices/stepDef/SoapStepDef.java

```diff
@@ -0,0 +1,70 @@
+package com.estefet.qapractices.stepDef;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.estefet.qapractices.request.CreateRequest;
+
+import cucumber.api.java.Before;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import io.restassured.RestAssured;
+import io.restassured.path.xml.XmlPath;
+import io.restassured.response.Response;
+
+/*
+ * Created by Anton Rusanov on 31/05/2018
+ */
+/*
+ * This is our step definition class for the cucumber steps.
+ */
+public class SoapStepDef {
```

Comment:
Same as in the other module:
1. Remove hardcoded values.
2. No 1's in the default tags.
3. Default method names.